src/App.js

```diff
@@ -1,12 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] As described by the name of this milestone 'Components' please make sure App.Js has class based component instead of simple function.